tests/utils/cardContent.js

```diff
@@ -0,0 +1,26 @@
+
+const playgroundUrlPlaceholders = '${PLAYGROUND_PROXY}/${CURRENT_HOSTNAME}'
+
+const getCardAMPContent = async (browser, cardSelector) => {
+    const contentInfo = await browser.executeAsync(async (cardSelector, done) => {
+        let amp = document.querySelector(`${cardSelector} article`).firstElementChild.shadowRoot.AMP
+        let info = {url: amp.url, title: amp.title};
+        done(info)
+    }, cardSelector); 
+
+    return contentInfo
+}
+
+const hasRightContentLoaded = async (browser, cardSelector, contentConfig) => {
+    const expectedUrl = contentConfig.url.includes(playgroundUrlPlaceholders) ? 
+    contentConfig.url.replace(playgroundUrlPlaceholders, '') :
+    contentConfig.url;
+    
+    const contentInfo = await getCardAMPContent(browser, cardSelector);
```

Comment:
why is contentInfo coupled to AMP? Should this method work for web and AMP?

Reply:
the card content is always an AMPDocument